scripts/paletteBuilder.js

```diff
@@ -0,0 +1,57 @@
+function initializeMathPalette() {
+    $.getJSON( "../data/palettes.json", function(palettes) {
+        $('#mathPalette').html(buildMathPalette(palettes));
+    });
+}
+
+function buildMathPalette(palettes) {
+    var result = "<h3 class=\"sr-only\">math input buttons</h3>";
```

Comment:
lowercase?

Reply:
Yes, lowercase is used to all labels and I decided not to change this.